src/components/Cashout/Cashout.tsx

```diff
@@ -92,6 +92,43 @@ export const Cashout = ({}) => {
         fetchAndSetCrossChainDetails()
     }, [])
 
+    // For testing - check if URL has screen param
+    useEffect(() => {
```

Comment:
Should this stil be in here if it was for testing ?

Reply:
yes! Its useful
peanut.to/cashout?screen=confirm

No reason not to, way easier  for future testing